docs/release-notes/0.10/release-0.10.md

```diff
@@ -0,0 +1,65 @@
+# ML.NET 0.10 Release Notes
+
+[ML.NET](https://aka.ms/mlnet) 0.10 brings us one step closer to the stable v1 release. We understand that the API surface has been changing rapidly and we deeply appreciate the amazing support from [ML.NET](https://aka.ms/mlnet) community. These changes are necessary for the support of the stable API for many years to come. In the upcoming 0.11, and 0.12 releases before we reach 1.0, we will continue on refining the API and improving documentation.
+
+We have also instrumented [code coverage](https://codecov.io/gh/dotnet/machinelearning) tools as part of our CI systems and will continue to push for stability and quality in the code.
+
+One of the milestones that we have acheived in this release is moving `DataView` into a new and separate assembly under `Microsoft.Data` namespace. For detailed documentation on `DataView` please take a look at [IDataView design principles](https://github.com/dotnet/machinelearning/blob/master/docs/code/IDataViewDesignPrinciples.md).
+
+### Installation
+
+ML.NET supports Windows, MacOS, and Linux. See [supported OS versions of .NET
+Core
+2.0](https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md)
+for more details.
+
+You can install ML.NET NuGet from the CLI using:
+```
+dotnet add package Microsoft.ML
+```
+
+From package manager:
+```
+Install-Package Microsoft.ML
+```
+
+### Release Notes
+
+Below are a few of the highlights from this release. There are many other improvements in the API.
+
+* DataView moved into a separate assembly and NuGet package 
+([#2220](https://github.com/dotnet/machinelearning/pull/2220))
+
+* Added support for returning multiple ranked results when scoring a multi-class classification model. 
+([#2250](https://github.com/dotnet/machinelearning/pull/2250))
+  - This allows to classify something into more than one category, for instance, assign a product to multiple categories, not just one
```

Comment:
I don't think this is quite correct. This would be "multi label" classification which (AFAIK) isn't supported by ML.NET (yet).

Reply:
edited. Thanks.,